From bda3c47ae85d8aba46b21e5a9207e73f7521b1c4 Mon Sep 17 00:00:00 2001
From: Jonathon Jongsma <jjongsma@redhat.com>
Date: Wed, 19 Jul 2017 14:33:00 -0500
Subject: [PATCH] Change default screenshot name to "Screenshot.png"

Since the code attempts to append ".png" to filenames without an
extension, it doesn't make much sense to have the default filename be
extensionless.  Including the extension on the default filename makes
things more straightforward.

Related: rhbz#1455832
---
 src/virt-viewer-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virt-viewer-window.c b/src/virt-viewer-window.c
index 8d05e52..ef40ec7 100644
--- a/src/virt-viewer-window.c
+++ b/src/virt-viewer-window.c
@@ -988,7 +988,7 @@ virt_viewer_window_menu_file_screenshot(GtkWidget *menu G_GNUC_UNUSED,
     image_dir = g_get_user_special_dir(G_USER_DIRECTORY_PICTURES);
     if (image_dir != NULL)
         gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER (dialog), image_dir);
-    gtk_file_chooser_set_current_name(GTK_FILE_CHOOSER (dialog), _("Screenshot"));
+    gtk_file_chooser_set_current_name(GTK_FILE_CHOOSER (dialog), _("Screenshot.png"));
 
     if (gtk_dialog_run(GTK_DIALOG (dialog)) == GTK_RESPONSE_ACCEPT) {
         char *filename;
-- 
2.13.6

